source/reviews/speed.txt

```diff
@@ -0,0 +1,149 @@
+individual author can write content. The speed of an individual is
+important, it's just not *as* important as the velocity of the entire team.
+
+When code reviews are slow, several things happen:
+
+-   **The velocity of the team as a whole is decreased.** Yes, the individual
+    who doesn't respond quickly to the review gets other work done. However, new
+    features and bug fixes for the rest of the team are delayed by days, weeks,
+    or months as each PR waits for review and re-review.
+-   **Authors start to protest the review process.** If a reviewer only
+    responds every few days, but requests major changes to the PR each time,
+    that can be frustrating and difficult for authors. Often, this is
+    expressed as complaints about how "strict" or "nitpicky" the reviewer
+    is being. If the reviewer requests the *same* substantial changes
+    (changes which really do improve content health), but responds *quickly*
+    every time the developer makes an update, the complaints tend to disappear.
+    **Most complaints about the code review process are resolved by making the
+    process faster.**
+-   **Content health can be impacted.** When reviews are slow, there is increased
+    pressure to allow developers to submit PRs that are not as good as they
+    could be. Slow reviews also discourage content cleanups, refactorings, and
+    further improvements to existing PRs.
+
+How Fast Should Code Reviews Be?
+--------------------------------
+
+If you are not in the middle of a focused task, **you should do a review
+shortly after it comes in.**
+
+**One business day is the maximum time it should take to respond** to a code
+review request (i.e., first thing the next morning).
+
+Following these guidelines means that a typical PR should get multiple rounds of
+review (if needed) within a single day.
+
+Speed vs. Interruption
+----------------------
+
+There is one time where the consideration of personal velocity trumps team
+velocity. **If you are in the middle of a focused task, such as writing content,
+don't interrupt yourself to do a code review.**
+Research has shown that it can take a long time for an author to get back into a
+smooth flow of work after being interrupted. Interrupting yourself while writing
+is actually *more* expensive to the team than making another author wait a bit
+for a review.
+
+Instead, wait for a break point in your work before you respond to a request for
+review. This could be when your current task is completed, after lunch,
+returning from a meeting, coming back from the restroom, etc.
+
+Fast Responses
+--------------
+
+When we talk about the speed of reviews, it is the *response* time that we
+are concerned with, as opposed to how long it takes a PR to get through the
+whole review and be accepted. The whole process should also be fast, ideally,
+but it's even more important for the *individual responses* to come quickly
+than it is for the whole process to happen rapidly.
```

Comment:
I think it should be both: as a potential reviewer, both mentioning you'll review it as well as completing the review is important. I think you should only pick it up when you're ready to review and that the review should not take long. Otherwise, it can turn into a long-running transaction during which nobody else that might be free will touch it.